CHANGES.rst

```diff
@@ -1,3 +1,6 @@
+Releases
+~~~~~~~~
+
```

Comment:
Please remove these three lines, and instead modify `docs/changes.rst`. I'm not sure, but I think we use some kind of automation to update this file upon release.

```
Change History
--------------

.. include:: ../CHANGES.rst
```

Reply:
Made this change. The `----` conflicts with a lower hierarchy level in CHANGES.rst, resulting in a garbled TOC tree on the index page, so retained `~~~~` here. Let me know if that doesn't work for some reason.